admin-dev/themes/new-theme/js/components/translatable-field.ts

```diff
@@ -24,6 +24,7 @@
  */
 
 import {EventEmitter} from './event-emitter';
+import ComponentsMap from './components-map';
```

Comment:
```suggestion
import {EventEmitter} from '@components/event-emitter';
import ComponentsMap from '@components/components-map';
```
Same here for event emitter, if we could use the namespaces everywhere it would be great